packages/app/src/app/overmind/actions.ts

```diff
@@ -94,15 +94,32 @@ export const modalClosed: Action = ({ state }) => {
   state.currentModal = null;
 };
 
-export const signInClicked: AsyncAction<{ useExtraScopes: boolean }> = (
-  { actions },
-  options
-) => actions.internal.signIn(options);
+export const signInClicked: Action<string | void> = ({ state }, redirectTo) => {
+  state.signInModalOpen = true;
+  state.redirectOnLogin = redirectTo || '';
```

Comment:
```suggestion
  state.redirectOnLogin = redirectTo;
```

Reply:
👍

Reply:
Doing that gives a type error even with the default value